packages/protocol/contracts/thirdparty/risczero/IRiscZeroVerifier.sol

```diff
@@ -0,0 +1,234 @@
+/// https://github.com/risc0/risc0-ethereum/blob/release-1.0/contracts/src/IRiscZeroVerifier.sol
+/// with slight modifications
```

Comment:
Could you please let me know what is the point bringing over the whole codebase from RiscZero ?

To me, brining over +6 files, does not make sense, because the "slight modifications" you are referring to is just formatting (`forge fmt`) - but please correct me if i'm wrong.

If we dont need to modify the code (which seems we dont), best way to install it as a package with `npm` or configure a `git` submodule, so we can minimize the diffs and depend on the frozen codebase of release 1.0.0 from RiscZero.

Reply:
basically they are fmt, useless code removing and solidity version upgrade, although now I can not remember why I changed the solidity version. not sure if they have a npm packge for it, git submodule maybe work, can try later.

Reply:
> basically they are fmt, useless code removing and solidity version upgrade, although now I can not remember why I changed the solidity version. not sure if they have a npm packge for it, git submodule maybe work, can try later.

`git submodule` works perfectly also, if we rely on that release  version, and no need to have custom code. Seems better imo - less code change in `taiko-mono`.

Reply:
Lets not use `git submodule`, I actually prefer upgrading the dependencies knowingly.

Reply:
Used pnpm package to keep it outside.